Source/santad/SNTNotificationQueue.h

```diff
@@ -21,6 +21,6 @@
 
 @property(nonatomic) MOLXPCConnection *notifierConnection;
 
-- (void)addEvent:(SNTStoredEvent *)event customMessage:(NSString *)message;
+- (void)addEvent:(SNTStoredEvent *)event customMessage:(NSString *)message url:(NSString *)url;
```

Comment:
Nit: We have interfaces using `andCustomURL`, `customURL` and `url`. It'd be nice to have these be more consistent (but don't think we should block on it if you don't have time right now).

Reply:
Done, I think.